### DISCUSSION TITLE
Marten 4.00-alpha9, Could not load file or assembly 'System.Runtime.Loader'  when using .NET 4.6.2

### DISCUSSION THREAD

**Discussion by ymqc** [JasperFx/marten#1798] [Q&A]
My project is winform using .Net Framework4.62, this error occur when using any Marten session, I found maybe caused by this [https://stackoverflow.com/questions/45741039/could-not-load-file-or-assembly-system-runtime-loader-for-adding-application-p](url).

**Comment by Hawxy** [comment]
Sorry, Marten v4 does not support .NET Framework. You can either stick with v3 or move your Winforms application to NET Core 3.1+.
